config/config.js

```diff
@@ -34,12 +34,12 @@ c.mongo.collection = env.MUNCHER_MONGODB_COLLECTION || 'muncher';
 c.mongo.creds      = {};
 
 // fix mongo location if trailing slash was omitted
-if (c.mongo.location[c.mongo.location.length-1] !== '/') {
+if (c.mongo.location[c.mongo.location.length - 1] !== '/') {
   c.mongo.location += '/';
 }
 
 // fs paths
-c.fs.base       = env.MUNCHER_BASEPATH || '/tmp/muncher/';
+c.fs.base       = env.MUNCHER_BASEPATH || '/tmp/o2r-muncher/';
```

Comment:
If you change this path here, you also need to change it in the `docker-compose` files. Also, if we change this, shouldn't it be entirely project-specific, and not contain the name of _one_ implementation? I sugget `/tmp/o2r/`.